Cargo.toml

```diff
@@ -23,16 +23,17 @@ codecov = { repository = "jonhoo/msql-srv", branch = "master", service = "github
 maintenance = { status = "experimental" }
 
 [dependencies]
-nom = "5"
-mysql_common = "0.22"
-byteorder = "1"
+nom = "7.0.0-alpha1"
```

Comment:
I'd like to not depend on an alpha version — could you stick with 6 here please?

Reply:
The current dependencies have been carefully selected to avoid dependency conflicts.

Reply:
I'm not sure I follow — why is an alpha dependency necessary here?

Reply:
I also prefer the stable version, but unfortunately. If you upgrade `mysql_common`,  whether nom 5 or 6, there will be dependency conflicts.

Reply:
for nom 6: error: failed to select a version for `funty`
for nom 5, sorry, nom 5 is okay. But this will lag behind the major releases. If you want, we can go back to nom5.

Reply:
Huh, that's very weird. What exactly is the error you get for `funty`?

Reply:
nom 7 is now available. We can skip this.